macros/WCTE.mac

```diff
@@ -97,7 +97,9 @@
 #/random/setSeeds 2 1
 #/WCSim/random/seed 4
 
-
+# Note: SetDetectorElement **must** come before SetDarkRate, if not, changes in the 
+# DarkRate will not be applied
+#/DarkRate/SetDetectorElement tank
```

Comment:
```suggestion
# Note: SetDetectorElement **must** come before all /DarkRate/ options (e.g. /DarkRate/SetDarkRate), if not, changes specified
# (e.g. in the DarkRate) will not be applied
# (Technically they will be applied to the OD PMT dark parameters, which don't exist in WCTE = no effect is seen)
/DarkRate/SetDetectorElement tank
```

Just generalising the comment a bit more, and also turning it on by default, so people don't miss it in the future